app/Http/Controllers/CampaignsController.php

```diff
@@ -70,4 +74,26 @@ public function show(Campaign $campaign, Request $request)
     {
         return $this->item($campaign);
     }
+
+    /**
+     * Updates a specific campaign
+     * PATCH /api/campaigns/:id
+     *
+     * @param Request $request
+     * @param  \Rogue\Models\Campaign  $campaign
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function update(Request $request, Campaign $campaign)
+    {
+        // Only allow an admin/staff to update.
```

Comment:
It might be best to remove this comment (in case we change the logic in the policy & it gets out of date).